flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableFactoryTest.java

```diff
@@ -339,6 +341,18 @@ public void testJdbcValidation() {
 				"The value of 'sink.max-retries' option shouldn't be negative, but is -1.")
 				.isPresent());
 		}
+
+		// connection.max-retry-timeout should be positive
```

Comment:
// connection.max-retry-timeout shouldn't be smaller than 1 second

Reply:
your description is more appropriate